lib/pod_builder/command/install_sources.rb

```diff
@@ -11,6 +11,8 @@ def self.call
 
         PodBuilder::prepare_basepath
 
+        argument_pods = OPTIONS.has_key?(:all) ? [] : ARGV.dup
```

Comment:
Do we need the ternary here?

```suggestion
        argument_pods = ARGV.dup
```